### PR TITLE
Add base layout

### PR DESCRIPTION
**Pull request by VladSolodjankin**

### CODE REVIEW

File: src/modules/BaseLayout/BaseLayout.module.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+@import '_colors.css';
+
+.app-layout {
```
Comment (EvgeniyKvasyuk) [inline src/modules/BaseLayout/BaseLayout.module.css:3]: `.layout`? Here and in  the same case too

File: src/App.tsx [hunk 2]
```diff
@@ -1,16 +1,26 @@
...
   return (
     <StyledEngineProvider injectFirst>
-      <Routes>
-        <Route element={<MainPage />} path="*" />
-      </Routes>
+      <Router>
+        <Routes>
+          <Route element={<BaseLayout />}>
+            <Route element={<MainPage />} path="/" />
+            <Route path="/Appointments" element={<Appointments />} />
```
[the remark this answers is not in the document]
Comment (VladSolodjankin) [reply src/App.tsx]: moved it to constants

File: src/modules/Footer/Footer.module.css [hunk 3]
```diff
@@ -0,0 +1,7 @@
+@import '_colors.css';
+
+.app-footer {
```
[the remark this answers is not in the document]
Comment (VladSolodjankin) [reply src/modules/Footer/Footer.module.css]: done

File: src/modules/BaseLayout/BaseLayout.module.css (hunk 1 above)
Comment (VladSolodjankin) [reply src/modules/BaseLayout/BaseLayout.module.css]: done

File: tsconfig.paths.json [hunk 4]
```diff
@@ -6,7 +6,9 @@
       "@common/*": ["src/common/*"],
       "@assets": ["src/assets"],
       "@assets/*": ["src/assets/*"],
-      "@modules/*": ["src/modules/*"]
+      "@modules/*": ["src/modules/*"],
+      "@constants/*": ["src/constants/*"],
```
Comment (EvgeniyKvasyuk) [inline tsconfig.paths.json:10]: There is `common` for that. Let's wont rise up size source structure

File: config/webpack.config.js [hunk 5]
```diff
@@ -276,6 +276,8 @@ module.exports = function (webpackEnv) {
         '@modules': path.resolve('src', 'modules'),
         '@common': path.resolve('src', 'common'),
         '@components': path.resolve('src', 'components'),
+        '@constants': path.resolve('src', 'constants'),
```
Comment (EvgeniyKvasyuk) [inline config/webpack.config.js:279]: I would'n have added this. Let's use common folder

File: src/modules/Footer/Footer.tsx [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import classNames from 'classnames/bind';
+
+import styles from './Footer.module.css';
+
+const cn = classNames.bind(styles);
+
+export function Footer() {
+  return <footer className={cn('footer')}>Footer</footer>;
```
Comment (EvgeniyKvasyuk) [inline src/modules/Footer/Footer.tsx:10]: locales

File: src/pages/Appointments/Appointments.tsx [hunk 7]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export function Appointments() {
+  return <div>Appointments</div>;
```
Comment (EvgeniyKvasyuk) [inline src/pages/Appointments/Appointments.tsx:4]: locales

File: src/pages/MainPage/MainPage.tsx [hunk 8]
```diff
@@ -1,19 +1,10 @@
...
 
 export function MainPage() {
   return (
     <div>
-      <Header />
-      <Routes>
-        <Route path="/article" element={<ArticlePage />} />
-        {/* <Route path="/anotherPage" element={<AnotherPage/>}/> */}
-      </Routes>
       <Link to="article">Link to article</Link>
```
Comment (EvgeniyKvasyuk) [inline src/pages/MainPage/MainPage.tsx:8]: locales

**Review by EvgeniyKvasyuk — APPROVED** [review]
Added a few comment about locales, should fix it later, but ok for now

**Pull request merged**